examples/trials/mnist-pytorch/fc_template.yml

```diff
@@ -0,0 +1,49 @@
+apiVersion: frameworkcontroller.microsoft.com/v1
+kind: Framework
+metadata:
+  name: pytorchcpu
+  namespace: default
+spec:
+  executionType: Start
+  retryPolicy:
+    fancyRetryPolicy: true
+    maxRetryCount: 2
+  taskRoles:
+  - name: worker
+    taskNumber: 1
+    frameworkAttemptCompletionPolicy:
+      minFailedTaskCount: 1
+      minSucceededTaskCount: 3
+    task:
+      retryPolicy:
+        fancyRetryPolicy: false
+        maxRetryCount: 0
+      podGracefulDeletionTimeoutSec: 1800
+      pod:
+        spec:
+          restartPolicy: Never
+          hostNetwork: false
+          containers:
+          - name: mnist-pytorch
+            image: msranni/nni:latest
+            command: ["python", "mnist.py"]
+            ports:
+            - containerPort: 5001
+            volumeMounts:
+            - name: frameworkbarrier-volume
+              mountPath: /mnt/frameworkbarrier
+            - name: data-volume
+              mountPath: /tmp/mount
+          serviceAccountName: frameworkbarrier
+          initContainers:
+          - name: frameworkbarrier
+            image: frameworkcontroller/frameworkbarrier
+            volumeMounts:
+            - name: frameworkbarrier-volume
+              mountPath: /mnt/frameworkbarrier
+          volumes:
+          - name: frameworkbarrier-volume
+            emptyDir: {}
+          - name: data-volume
+            persistentVolumeClaim:
+              claimName: nni-storage
```

Comment:
Does users need to create PVC manually before create NNI experiments?

Reply:
Yes. I added some documentation for the new custom capability and explicitly mentioned this prerequisite. I'm aware of the
fact, that the k8s API offers and endpoint to create a PVC algorithmically, but this can easily mess up a cluster's storage management, so this seems like a bad idea. If a person is willing to use a PVC, then I guess it should be somewhat enforced this person has sufficient rights to do so and is aware what's about to happen, which makes me think manual creation beforehand is a good idea. What do you think?

Reply:
Agree to let users manually create PVC, as long as we declare it in doc and examples.